source/blog/2018-03-08-update-kernel-arguments-on-atomic-host.html.md

```diff
@@ -10,72 +10,72 @@ tags:
-post also might be a bit difficult to understand without prior knowledge to
-rpm-ostree and Atomic Host, please bear that in mind when reading this.
-
+post also requires some knowledge of Atomic Host and rpm-ostree, please
+bear that in mind when reading this.
 
 READMORE
 
 Let's demonstrate some of the options that can be done with this command!
 
 # Display the current kernel arguments
 
-The rpm-ostree ex kargs command can allow users to see the kernel arguments
+The `rpm-ostree ex kargs` command can allow users to see the kernel arguments
 from different locations, including the current kernel arguments, arguments
-from /proc/cmdline as well as kernel arguments from a specific deployment
+from /proc/cmdline as well as kernel arguments from a specific deployment.
 
 
 The command following will show the kernel arguments from first bootable entry
 in your grub.cfg:
+
 ```
 [root@localhost ~]# rpm-ostree ex kargs
 The kernel arguments are:
 no_timer_check console=tty1 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 rd.lvm.lv=atomicos/root root=/dev/mapper/atomicos-root ostree=/ostree/boot.0/fedora-atomic/9a9b350be75846811cbb0b1fd7b3d42a49908ed1265bc59e292bb4a34674332c/0
 ```
 
-This command will show you the kernel arguments from the first deployment
+This command will show you the kernel arguments from the first deployment:
+
 ```
 [root@localhost ~]# rpm-ostree ex kargs --deploy-index=0
 The kernel arguments are:
 no_timer_check console=tty1 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 rd.lvm.lv=atomicos/root root=/dev/mapper/atomicos-root ostree=/ostree/boot.0/fedora-atomic/9a9b350be75846811cbb0b1fd7b3d42a49908ed1265bc59e292bb4a34674332c/0
 ```
 
-This will show you the current kernel arguments being used in /proc/cmdline
+This will show you the current kernel arguments being used in /proc/cmdline:
 ```
 [root@localhost ~]# rpm-ostree ex kargs  --import-proc-cmdline
 The kernel arguments are:
 no_timer_check console=tty1 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0 rd.lvm.lv=atomicos/root root=/dev/mapper/atomicos-root ostree=/ostree/boot.0/fedora-atomic/9a9b350be75846811cbb0b1fd7b3d42a49908ed1265bc59e292bb4a34674332c/0
-
 ```
 
 # Changing the kernel arguments
 
 Similarly, you can fetch kernel arguments from three different locations mentioned above,
-but for readability and simplicty, we are only demoing with the first one(first
+but for readability and simplicty, we are only demoing with the first one (first
 bootable entry) here.
 
-You have four different options of changing kernel arguments in the command, all of those
-will create a deployment, meaning its changes can be revertable. A more detailed description
-will be shown by `rpm-ostree ex kargs --help`
+You have four different options of changing kernel arguments in the command. All of those
+will create a deployment, and can be revertible through `rpm-ostree rollback`. A more
+detailed description of the commands will be shown by `rpm-ostree ex kargs --help`
```

Comment:
nit: "will be shown" sounds like you are going to show the output of the command. Maybe "can be shown"?

Reply:
yup, noticed that too, done so in the following fixup :).